aptos-move/framework/aptos-stdlib/sources/fixed_point64.move

```diff
@@ -0,0 +1,293 @@
+/// Defines a fixed-point numeric type with a 64-bit integer part and
+/// a 64-bit fractional part.
+
+module aptos_std::fixed_point64 {
+
+    /// Define a fixed-point numeric type with 64 fractional bits.
+    /// This is just a u128 integer but it is wrapped in a struct to
+    /// make a unique type. This is a binary representation, so decimal
+    /// values may not be exactly representable, but it provides more
+    /// than 9 decimal digits of precision both before and after the
+    /// decimal point (18 digits total). For comparison, double precision
+    /// floating-point has less than 16 decimal digits of precision, so
+    /// be careful about using floating-point to convert these values to
+    /// decimal.
+    struct FixedPoint64 has copy, drop, store { value: u128 }
+
+    const MAX_U128: u256 = 340282366920938463463374607431768211455;
+
+    /// The denominator provided was zero
+    const EDENOMINATOR: u64 = 0x10001;
```

Comment:
Are you using hex values here to be consistent with fixed_point32?

Reply:
yes